src/builtin/numbers.scrbl

```diff
@@ -361,10 +374,12 @@ the same line denote the same Pyret number.
 -6.022e-23
 }
 
-Exact numbers are of arbitrary precision.
+@pyret{Exactnum}s are of arbitrary precision.
+
+@pyret{Roughnum}s are represented with a leading tilde.  You can think of
+the tilde as representing a person waving his or her hands vaguely.
 
-Literal roughnums are represented with a leading tilde. They are
-either integers or decimals, with an optional exponent.
+They are either integers or decimals, with an optional exponent.
```

Comment:
There exist roughnum fractions, too.  You can _write_ `~1/2`, and it'll produce `~0.5`.  It will never render as `~1/2`, though.